start-deployment-manager.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")"
+
+if [ "$#" -ne 2 ]; then
+    echo "Wrong number of arguments. Usage: <Assembly postfix> <deployment manager project directory>"
+fi
+
+ASSEMBLY_POSTFIX="$1"
+LAUNCH_CONFIGURATION="$2"
+
+echo ${ASSEMBLY_POSTFIX}
+
+cp ${LAUNCH_CONFIGURATION} workers/deployment_manager/DeploymentManager/default_launch.json
+
+pushd "workers/deployment_manager"
+  sh publish-linux-workers.sh
+popd
+
+spatial build build-config
+spatial cloud upload "deployment_manager_${ASSEMBLY_POSTFIX}" --force
```

Comment:
Given the character limits on the assembly name - I would recommend shortening the automatic prefix to lower the chance of rejection. 

I.e. - `deployment_manager_${ASSEMBLY_POSTIFX}` --> `depl_mng_${ASSEMBLY_POSTFIX}` or similar

Reply:
might as well just change it completely and use `ASSEMBLY_NAME` instead of `ASSEMBLY_POSTFIX` as it seems to confuse people ;)